raft.go

```diff
@@ -427,6 +430,9 @@ type raft struct {
 	// current term. Those will be handled as fast as first log is committed in
 	// current term.
 	pendingReadIndexMessages []pb.Message
+
+	traceLogger     TraceLogger
+	initStateTraced bool
```

Comment:
It seems that we can remove `initStateTraced`, and only depend on `traceLogger` to check whether the trace is enabled or not?

Reply:
correct. I'll remove this member.

Reply:
Fixed